samples/NativeLibrary/c_source/readme.txt

```diff
@@ -0,0 +1 @@
+MAKE SURE TO COMPILE WITH -LDL FLAG
```

Comment:
May this can be just a comment in main.c.  Also:
- This is applicable to Unix only (prefix it with, "On Unix, make sure ....")
- `-LDL` should be lower-case. It won't work as upper case.